src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_process_net_command.py

```diff
@@ -257,9 +257,20 @@ def cmd_set_break(self, py_db, cmd_id, seq, text):
         filename = self.api.filename_to_server(filename)
         func_name = self.api.to_str(func_name)
 
-        self.api.add_breakpoint(
+        error_code = self.api.add_breakpoint(
             py_db, filename, btype, breakpoint_id, line, condition, func_name, expression, suspend_policy, hit_condition, is_logpoint)
 
+        if error_code:
+            if error_code == self.api.ADD_BREAKPOINT_FILE_NOT_FOUND:
+                pydev_log.critical('pydev debugger: warning: Trying to add breakpoint to file that does not exist: %s (will have no effect).' % (filename,))
+
+            elif error_code == self.api.ADD_BREAKPOINT_FILE_EXCLUDED_BY_FILTERS:
+                pydev_log.critical('pydev debugger: warning: Trying to add breakpoint to file that is excluded by filters: %s (will have no effect).' % (filename,))
```

Comment:
I think if we can say what filter is excluding this file, it might be helpful. Especially when `rules` starts to become a thing. This is not blocking, but if we can add that it would really be helpful.

Reply:
I'll take a look at that.

Reply:
I started to look and it'd change more than I'd like for this pull request, so, I created https://github.com/microsoft/ptvsd/issues/1464 to address this.